domenetjenester/iay/src/main/java/no/nav/foreldrepenger/abakus/iay/tjeneste/ArbeidsforholdRestTjeneste.java

```diff
@@ -76,12 +76,13 @@ public ArbeidsforholdRestTjeneste(KoblingTjeneste koblingTjeneste, InntektArbeid
     public Response hentArbeidsforhold(@NotNull @TilpassetAbacAttributt(supplierClass = AktørDatoRequestAbacDataSupplier.class) @Valid AktørDatoRequest request) {
         AktørId aktørId = new AktørId(request.getAktør().getIdent());
         Periode periode = request.getPeriode();
+        Fagsystem innhenter = request.getFagsystem() != null ? request.getFagsystem() : Fagsystem.K9SAK;
```

Comment:
vi bruker ytelsetype som input ellers - så hvorfor ikke bruke det i stedet for fagsystem?  fagsystem kan fort endre seg hvis man deler opp tjenester

Reply:
Kan godt bytte ut den. Det er ikke så tydelig ettersom det her spørres fra kompletthetssjekken

Reply:
tenker det gjør apiet mer konsistent?

Reply:
Ytelsetype er mest konsistent. Alternativet er antagelig Tema (FOR/OMS)